extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts

```diff
@@ -706,9 +706,9 @@ export class PublishDatabaseDialog {
 		}).component();
 
 		loadSqlCmdVarsButton.onDidClick(async () => {
-			for (const varName in this.sqlCmdVars) {
+			for (const [key, _value] of this.sqlCmdVars!) {
```

Comment:
since you don't need `_value`, you could also do `for (const key of this.sqlCmdVars.keys()) {...}`